lib/safira_web/controllers/auth_controller.ex

```diff
@@ -11,7 +11,8 @@ defmodule SafiraWeb.AuthController do
     with {:ok, multi} <- Auth.create_user_uuid(user_params),
          {:ok, token, _claims} <- Guardian.encode_and_sign(multi.user) do
       conn
-      |> render("jwt.json", jwt: token)
+      |> render("signup_response.json", %{jwt: token,
+      discord_association_code: multi.attendee.discord_association_code})
```

Comment:
Is it okay if I access a nested map this way, instead of having to user Map.get twice?

Reply:
@0urobor0s @ZePedroResende What do you think of this? It's okay?

Reply:
You could this in the with

Reply:
Pattern matching in the multi, I see. Nice :+1:

Reply:
But to pattern match wouldn't I have to specify the whole format of the `multi` variable?

Reply:
`{attendee: {discord_association_code: code}} <- multi`
 Use `code` as the variable

Reply:
I think @ZePedroResende meant `{:ok, %{attendee: %{discord_association_code: code}}} <- Auth.create_user_uuid(user_params)`

Here is a quick test:

![image](https://user-images.githubusercontent.com/19409687/106519110-ba832a00-64d2-11eb-8f97-d9101f311e5c.png)

As you can see, I am ignoring age.

You could also do `Auth.create_user_uuid(user_params).attendee.discord_association_code`, but I prefer the first option.

Reply:
Sorry @ZePedroResende, didn't saw you comment :facepalm:

Reply:
I was going to change the code in order to use this pattern matching but I just noticed that the `multi` variable is also used in the second life of the with statement (`{:ok, token, _claims} <- Guardian.encode_and_sign(multi.user)`). So, I had to change the switch statement a bit to 
`with {:ok, %{user: user,
          attendee: %{discord_association_code: code}}} <- Auth.create_user_uuid(user_params),
         {:ok, token, _claims} <- Guardian.encode_and_sign(user)`
         
 Is that okay?